src/storage/mvcc/reader/scanner/forward.rs

```diff
@@ -689,19 +689,18 @@ where
     }
 }
 
-#[cfg(test)]
-mod test_util {
+pub mod test_util {
```

Comment:
What about adding a new feature configuration for this?

Reply:
I really no want to add a new feature because of one single mod. I think TiKV's features are too complex 😕

Reply:
I think it's a common requests to expose test utils for other module. The features may be reused in the future. Besides it's also beneficial for release build.